stingray/lightcurve.py

```diff
@@ -451,8 +451,8 @@ def join(self, other):
             except IndexError:
                 count2 = None
 
-            if not count1 is None:
-                if not count2 is None:
+            if count1 is not None:
+                if count2 is not None:
```

Comment:
Oh yes!

In this context it does also work because `None` evaluates to `False` in a comparison, then is mostly a Style choice.

Usually I prefer to be explicitly in testing values. Can we be sure that whatever value it is it will be evaluated as expected? (e.g, a list with None values is evaluated to `True` while an empty list is evaluated  to `False`)

I can change if you wanted, but for me the explicit check is better :)